Cloud destination updates (#774)
rudderlabs/rudderstack-docs: docs/destinations/overview.mdx

---
title: "Destinations Overview"
description: >-
  Detailed technical documentation on sending event data from RudderStack to your preferred destination platforms.
---

## What is a destination?

A RudderStack destination is the target for your source event data, generally either a **cloud** destination or a **warehouse** destination. 

* **Cloud destinations** consume your event data to complement their offering related to marketing, analytics, CRM, etc.
* **Warehouse destinations** are data storage and processing platforms like Amazon Redshift, Snowflake, or Google BigQuery.

RudderStack currently supports integration with over 150 cloud and warehouse destinations. This section provides detailed documentation on setting up these destinations in RudderStack to send your event data in no time.

## Warehouse Destinations

<Destinations
  type={'warehouse'}
/>

## Cloud Destinations

<Destinations
  type={'streaming'}
/>

<div class="infoBlock">

> To send events to a destination that RudderStack currently does not support or to build a new integration for RudderStack, refer to the <Link to="/resources/community/#contribute">Contributor's guides</Link>.

Reviewer: ```suggestion
To send events to a destination that RudderStack does not support currently, you can build a new integration for that destination by referring to the <Link to="/resources/community/#contribute">Contributor's guides</Link>.
```
